--- configure.in	2008-12-23 21:10:32.000000000 +0100
+++ configure.in.oden	2008-12-23 21:10:39.000000000 +0100
@@ -47,6 +47,11 @@
 
 AC_CHECK_LIB(m,floor)
 
+AC_CHECK_FUNCS([floor])
+if test $ac_cv_func_floor = no; then
+  AC_CHECK_LIB(m, floor, [LIBS="$LIBS -lm"])
+fi
+
 AC_PATH_PROG(GZIP,gzip,,[/usr/bin:/bin:/usr/local/bin:/opt/bin:$PATH])
 if test -z "$GZIP"; then
     echo "Could not locate gzip"
